docs/_tutorials/request-reply.md

```diff
@@ -202,24 +197,23 @@ All that’s left is to receive and process the reply message as it is received
 
 ```java
 if (reply.getJMSCorrelationID() == null) {
-    throw new Exception("Received a reply message with no correlationID. This field is needed for a direct request.");
-}
-
-//The reply's correlationID should match the request's correlationID
-if (!reply.getJMSCorrelationID().equals(correlationId)) {
-    throw new Exception("Received invalid correlationID in reply message.");
-}
-
-if (reply instanceof TextMessage) {
-    System.out.printf("TextMessage response received: '%s'%n",
-        ((TextMessage)reply).getText());
-    if     (!reply.getBooleanProperty(SupportedProperty.SOLACE_JMS_PROP_IS_REPLY_MESSAGE)) {
-
-        System.out.println("Warning: Received a reply message without the isReplyMsg flag set.");
+    System.out.println(
+            "Received a reply message with no correlationID. This field is needed for a direct request.");
+} else {
```

Comment:
Mark, I'm not sure I understand how you got to this conclusion, I'd suggest we look at it tomorrow morning together, please drop by at the time of your convenience.